compiler-rt/lib/scudo/standalone/memtag.h

```diff
@@ -66,7 +66,8 @@ inline bool systemSupportsMemoryTagging() {
 #ifndef HWCAP2_MTE
 #define HWCAP2_MTE (1 << 18)
 #endif
-  return getauxval(AT_HWCAP2) & HWCAP2_MTE;
+  static bool SupportsMemoryTagging = getauxval(AT_HWCAP2) & HWCAP2_MTE;
```

Comment:
Oh,
my thought during review: "nice! static locals are supported now" :)

https://lab.llvm.org/buildbot/#/builders/24/builds/14562

Reply:
Memory tagging support status is deliberately cached in the UseMemoryTagging bit instead of here to avoid issues with static locals such as those pointed out by the buildbot.